src/sass/_slider.scss

```diff
@@ -0,0 +1,30 @@
+.carousel__img {
+  min-width: 100%;
+  object-fit: cover;
+  height: 300px;
+}
+
+.carousel-item {
+  height: 300px;
+
+  &::after {
+    display: block;
+    position: relative;
+    background-image: linear-gradient(
+      to bottom,
+      rgba(($color-primary), 0.1) 0,
+      rgb(197, 16, 16) 100%
+    );
+    margin-top: -300px;
+    height: 100%;
+    width: 100%;
+    content: "";
+  }
+}
+
+.carousel {
+  &__caption-wrapper {
+    background-color: rgba(($color-primary), 0.6);
+    border-radius: 5px;
```

Comment:
`@extend .rounded` Could be sufficent
https://getbootstrap.com/docs/4.0/utilities/borders/#border-radius